apps/web/src/app/components/CollateralTypeSwitcher/CollateralTypeList.tsx

```diff
@@ -1,207 +1,164 @@
-  ${({ $negativeMargin }) => ($negativeMargin ? 'margin-top: -10px;' : '')}
-  transition: transform ease .3s;
-
-  .white,
-  .grey {
-    transition: all ease-out 0.2s;
-  }
-
-  .white {
-    color: #ffffff;
-  }
-
-  .grey {
-    color: #d5d7db;
-  }
-
-  &:hover {
-    .white,
-    .grey {
-      color: ${({ theme }) => theme.colors.primaryBright};
-    }
-  }
-`;
-
-const StyledSkeleton = styled(Skeleton)`
-  margin-left: auto;
-`;
-
-const CollateralTypesGridHeader = styled.div`
-  text-transform: uppercase;
-  text-align: right;
-  font-size: 14px;
-  letter-spacing: 3px;
-  padding-bottom: 10px;
-  color: ${({ theme }) => theme.colors.text1};
-
-  &:first-of-type {
-    text-align: left;
-  }
-`;
-
-const CollateralTypesGridItem = styled.div`
-  text-align: right;
-  font-size: 14px;
-  padding: 20px 0 20px 10px;
-  cursor: pointer;
-
-  &:first-of-type {
-    text-align: left;
-    padding: 20px 0;
-  }
-`;
-
-const AssetInfo = styled.div`
-  display: flex;
-  position: relative;
-  align-items: center;
-  padding-left: 40px;
-
-  img {
-    position: absolute;
-    left: 0;
-  }
-
-  @media screen and (max-width: 350px) {
-    padding-left: 0;
-    img {
-      display: none;
-    }
-  }
-`;
-
-const GridWrap = styled.div`
-  max-height: 305px;
-  overflow-y: auto;
-  padding: 0 25px;
-`;
-
-const CollateralTypeList = ({ width, setAnchor, anchor, ...rest }) => {
+import { useLoanActionHandlers, useSetLoanRecipientNetwork } from 'store/loan/hooks';
+import { Trans } from '@lingui/macro';
+import { BalanceAndValueWrap, DashGrid, HeaderText, List, walletBreakpoint } from '../Wallet/styledComponents';
+import SingleChainItem from './SingleChainItem';
+import MultiChainItem from './MultiChainItem';
+import { CollateralTab } from './CollateralTypeListWrap';
+import { ICON_XCALL_NETWORK_ID } from 'constants/config';
+import SingleChainItemOverview from './SingleChainItemOverview';
+import { xChainMap } from 'app/pages/trade/bridge/_config/xChains';
+import { Typography } from 'app/theme';
+import { XChainId } from 'app/pages/trade/bridge/types';
+
+const CollateralTypeList = ({
+  setAnchor,
+  collateralTab,
+  query,
+}: {
+  setAnchor: (anchor: HTMLElement | null) => void;
+  collateralTab: CollateralTab;
+  query: string;
+}) => {
   const changeCollateralType = useCollateralChangeCollateralType();
-  const allCollateralData = useAllCollateralData();
-  const hideCollateralInfoColumn = useMedia('(max-width: 500px)');
-  const handleEscape = useKeyPress('Escape');
+  const setLoanNetwork = useSetLoanRecipientNetwork();
+  const changeCollateralXChain = useChangeCollateralXChain();
   const { onAdjust: adjust } = useCollateralActionHandlers();
   const { onAdjust: adjustLoan } = useLoanActionHandlers();
+  const isSmallScreen = useMedia(`(max-width: ${walletBreakpoint})`);
+
+  const { data: allPositionsData } = useXCollateralDataByToken();
+  const { data: allCollateralData } = useAllCollateralData();
```

Comment:
rename to allPositionData

Reply:
moved to https://github.com/balancednetwork/balanced-network-interface/issues/1388